archive/tar_windows.go

```diff
@@ -26,21 +26,6 @@ import (
 	"github.com/containerd/containerd/sys"
 )
 
-// tarName returns platform-specific filepath
-// to canonical posix-style path for tar archival. p is relative
-// path.
-func tarName(p string) (string, error) {
-	// windows: convert windows style relative path with backslashes
-	// into forward slashes. Since windows does not allow '/' or '\'
-	// in file names, it is mostly safe to replace however we must
-	// check just in case
-	if strings.Contains(p, "/") {
-		return "", fmt.Errorf("windows path contains forward slash: %s", p)
-	}
```

Comment:
This got removed in https://github.com/moby/moby/commit/a5aed699cfaa4d84b1b134033fb468b3a7a874f0 https://github.com/moby/moby/pull/37356#discussion_r198622023